src/components/KSelect/KSelect.vue

```diff
@@ -40,14 +40,14 @@
           </div>
         </slot>
         <button
-          class="clear-selection-icon cursor-pointer non-visual-button"
+          class="clear-selection-icon"
           @click="clearSelection"
           @keyup.enter="clearSelection"
         >
           <KIcon
             color="var(--blue-200)"
             icon="clear"
-          />
+          /><!-- TODO: token needed -->
```

Comment:
@adamdehaven any idea what can we do here? Appropriate `color-text-primary-...` token variant doesn't exist and `$tmp-` SCSS won't work in template (unless used with interpolation?)

Reply:
You're saying there's not a token for `blue-200`, correct?

If that's the case, I would pick the _closest existing new `kui` token_ and use that, with its fallback, instead. 

Changing the icon color a little isn't going to break anything.

Reply:
There is no close value color text token (closest one is `color-text-primary` which is noticeably different). There are close tokens for background color, but we decided against using background color tokens for icons color. I would add a temp SCSS token for this so we can decide later, but like I said it won't work.

Reply:
Just use the closest and put a screenshot in the PR

Reply:
Here are before and after screenshots
![Screen Shot 2023-07-18 at 11 13 25 AM](https://github.com/Kong/kongponents/assets/36751160/50384476-e2bf-4f7f-b34b-86ab8ca362ed)
![Screen Shot 2023-07-18 at 11 16 17 AM](https://github.com/Kong/kongponents/assets/36751160/5ef6f61b-08f2-4e35-9d4c-e0e451ca9f5d)

Reply:
I think the easiest thing for this particular scenario is to hard-code the icon color to the hex value and just drop CSS variable support. I don’t believe we’re customizing he select icon color here anyway